utils/file_it_test.go

```diff
@@ -75,9 +75,9 @@ func testWatchWatcherEvents(t *testing.T) {
 		Op:   fsnotify.Create,
 	}
 	stopWatching := make(chan struct{}, 1)
-	f := func(itmPath, itmID string) error {
+	f := func(itmID string) error {
```

Comment:
Will this not create extra files in the git monitored repo? Temporary folder was there to prevent that.